packages/ui-core/src/components/table/table.styles.ts

```diff
@@ -0,0 +1,63 @@
+import styled, { css } from 'styled-components';
+
+export const Sticky = styled.div`
+  display: flex;
+  flex-direction: row;
+  padding: 20px;
+`;
+
+export const StyledHeaderCeil = styled.div<{
+  width: number;
+}>`
+  color: #fff;
+  font-size: 14px;
+  line-height: 17px;
+  text-align: left;
+  font-family: Gangster Grotesk Bold;
+  white-space: nowrap;
+
+  ${props =>
+    props.width &&
+    css`
+      width: ${props.width}px;
+    `}
+`;
+
+export const StyledCeil = styled.div<{
+  type: string;
```

Comment:
The pure visualization component should not know the `type` of value. IMO it should accept some kind of positioning property like for eg. `textAlign` that could be controlled.